db/seeds.rb

```diff
@@ -1,5 +1,32 @@
 locations = {
   meetups: [
+    {
+      slug: 'ruby-tuesday-tallinn',
+      name: 'Ruby Tuesday Tallinn',
+      description: 'Ruby User Group meeting every last Tuesday of the month from 18:00 onward.',
+      latitude: 59.398279,
+      longitude: 24.659693,
+      link: 'https://www.facebook.com/events/785659903522120',
+      img: './ruby-tuesday-tallinn.jpg'
```

Comment:
This should link to public/logos - since you've put (great!) the logos there. So it should be `/logos/ruby-tuesday-tallinn.jpg` - just like in https://github.com/rubyeurope/rubyeurope.com/pull/1/files#diff-c61a903a1201b0c7f076b0270377c20390c26f655ce63689d7d843296fcf63f3R287